helm/admin-service/values.yaml

```diff
@@ -190,7 +190,16 @@ podAnnotations: {}
 
 ## lifecycleHooks for the  container to automate configuration before or after startup.
 ##
-lifecycleHooks: {}
+lifecycleHooks:
+  preStop:
+    exec:
+      command:
+        - sh
+        - -c
+        - sleep 30
+
+## Termination grace perios : the maximum amount of time (in seconds) Kubernetes will wait for a container to gracefully shut down
+terminationGracePeriodSeconds: 60
 ## Custom Liveness probes for
 ##
 customLivenessProbe: {}
```

Comment:
repository: bitnami/bitnami-shell
tag: "10" 
to
repository: mosipid/os-shell
tag: "12-debian-12-r46"
	
and 
tag: 1.3.0-beta.2 
to 
tag: 1.3.x